main-project/src/chats/chats-gateway.service.ts

```diff
@@ -102,47 +82,55 @@ export class ChatsGatewayService {
     return { chatRoomNo, roomName };
   }
 
-  private async setChatRoom(
+  private async setChatRoomUsers(
     manager: EntityManager,
-    chatRoomUsers: ChatRoomUsers,
+    chatRoomUsers: ChatRoomWithUsers,
   ): Promise<void> {
-    const { userType, chatRoomNo }: ChatRoomUsers = chatRoomUsers;
+    const { userType, chatRoomNo }: ChatRoomWithUsers = chatRoomUsers;
     const users = chatRoomUsers.users.split(',').map(Number);
 
-    const chatUsers: ChatUserInfo[] = users.reduce((values, userNo) => {
+    const chatUsers: ChatUser[] = users.reduce((values, userNo) => {
       values.push({ chatRoomNo, userNo, userType });
 
       return values;
     }, []);
 
-    await this.setChatRoomUsers(manager, chatUsers);
+    await this.createChatUsers(manager, chatUsers);
   }
 
-  private async checkChatRoomExists(boardNo): Promise<void> {
-    const board: Board = await this.boardRepository.getBoardByNo(boardNo);
-    if (!board.no) {
-      throw new NotFoundException(`게시물을 찾지 못했습니다.`);
+  private async checkChatRoomExists(
+    boardNo: number,
+    userNo: number,
+  ): Promise<void> {
+    const board: Boards = await this.boardRepository.getBoard(boardNo);
```

Comment:
```suggestion
    const board: Board = await this.boardRepository.getBoardByNo(boardNo);
```
boardrepo에선 getBoard함수가 없습니다!